core/src/main/scala/org/broadinstitute/dsde/rawls/openam/StandardUserInfoDirectives.scala

```diff
@@ -1,17 +1,13 @@
 package org.broadinstitute.dsde.rawls.openam
 
 import org.broadinstitute.dsde.rawls.model.{RawlsUserEmail, RawlsUserSubjectId, UserInfo}
-import org.broadinstitute.dsde.vault.common.util.ImplicitMagnet
 import spray.http.{HttpHeader, OAuth2BearerToken}
 import spray.routing.Directive1
 import spray.routing.Directives._
 
-import scala.concurrent.ExecutionContext
-
 trait StandardUserInfoDirectives extends UserInfoDirectives {
 
-  def requireUserInfo(magnet: ImplicitMagnet[ExecutionContext]): Directive1[UserInfo] = {
-    implicit val ec = magnet.value
```

Comment:
This is the only place `magnet` was used, but we do not need to provide an `ExecutionContext` to anything here.